libraries/chain/include/eosio/chain/fork_database.hpp

```diff
@@ -86,6 +87,12 @@ namespace eosio::chain {
        */
       branch_type fetch_branch( const block_id_type& h, uint32_t trim_after_block_num = std::numeric_limits<uint32_t>::max() ) const;
 
+      /**
+       *  Returns full branch of `block_header_state` pointers all the way to the fork_db root.
```

Comment:
I think this should say "Returns full branch of `block_header_state` pointers including the `root`". It was not clear to me that this included the root.

Reply:
sure!